packages/expo-updates/ios/EXUpdates/Update/EXUpdatesLegacyUpdate.m

```diff
@@ -47,8 +47,7 @@ + (EXUpdatesUpdate *)updateWithLegacyManifest:(EXUpdatesLegacyRawManifest *)mani
   NSString *bundleUrlString = manifest.bundleUrl;
   NSArray *assets = manifest.bundledAssets ?: @[];
   
-  // Use the manifest's runtimeVersion if it is not null AND the config's runtimeVersion is not null
-  if (manifest.runtimeVersion != nil && config.runtimeVersion != nil) {
+  if (manifest.runtimeVersion != nil) {
```

Comment:
Similarly, do we want to change this logic? It seems like if a client is not configured with a custom runtime version (Expo Go, standalone apps), it should prefer the SDK version field instead.

Reply:
mentioned above in detail, but if a client is not configured with a custom runtime version, it should be sending the `expo-sdk-version` header and getting back manifests that do not include a runtimeVersion.